App/UIElements/SimFrame.cs

```diff
@@ -66,7 +68,7 @@ private void OnEntityAdded(Entity entity)
         
         private void OnEntityRemoved(Entity entity)
         {
-            foreach (IRenderer renderer in _renderers)
+            foreach (IRenderer renderer in _renderers.ToList())
             {
                 if (!(renderer is EntityRenderer entityRenderer)) continue;
                 
```

Comment:
wouldn't this mean a sim frame in a sim frame wouldn't work?

Reply:
Yep, I don't think it is logical/useful since SimFrame is tied to a World. What do you think ?

Reply:
Yep seems logical since we don't create a world with a sim world. Just means we won't be able to create two side-by-side simulations though

Reply:
It is in fact possible since simframes are still able to be attached to the main frame